packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -476,7 +474,71 @@ export default class WcSdk implements Neo3Invoker, Neo3Signer {
         return resp as NetworkVersion;
     }
 
-    private validateContractInvocationMulti (request: ContractInvocationMulti): boolean {
+    async encrypt(message: string, publicKeys: string[]): Promise<EncryptedPayload[]> {
+        const request = {
+            id: 1,
+            jsonrpc: "2.0",
+            method: "encrypt",
+            params: [message, publicKeys]
+        }
+        const resp = await this.signClient.request({
+            topic: this.session?.topic ?? "",
+            chainId: this.getChainId() ?? "",
+            request
+        })
+
+        if (!resp) {
+            throw new WcSdkError(resp);
+        }
+
+        return resp as EncryptedPayload[]
+    }
+    async decrypt(payload: EncryptedPayload): Promise<string> {
+        const request = {
+            id: 1,
+            jsonrpc: "2.0",
+            method: "decrypt",
+            params: [payload]
+        }
+        const resp = await this.signClient.request({
+            topic: this.session?.topic ?? "",
+            chainId: this.getChainId() ?? "",
+            request
+        })
+
+        if (!resp) {
+            throw new WcSdkError(resp);
+        }
+
+        return resp as string
+    }
+    async decryptFromArray(payloads: EncryptedPayload[]): Promise<{

```

Comment:
Pula linha aqui tb pf
```suggestion

    async decryptFromArray(payloads: EncryptedPayload[]): Promise<{

```

Reply:
coloquei o espaçamento entre os métodos